### PR TITLE
Fix configuration bugs

### PR DESCRIPTION
**Pull request by nopara73**

### CODE REVIEW

**Comment by bokobza** [comment]
This looks good.
I'll wait for the builds to finish and I'll check it in.

File: Stratis.Bitcoin/Configuration/NodeSettings.cs [hunk 1]
```diff
@@ -368,12 +369,20 @@ public Network GetNetwork()
...
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
-                this.Logger.LogInformation("Using HOME environment variable for initializing application data.");
-                directory = home;
-                directory = Path.Combine(directory, "." + appName.ToLowerInvariant());
+                var home = Environment.GetEnvironmentVariable("HOME");
+                if (!string.IsNullOrEmpty(home))
+                {
+                    this.Logger.LogInformation("Using HOME environment variable for initializing application data.");
+                    directory = home;
```
Comment (bokobza) [inline Stratis.Bitcoin/Configuration/NodeSettings.cs]: directory = home; this seems like an extra assignment in vain.
why not do 
` directory = Path.Combine(home, "." + appName.ToLowerInvariant());`
same for localAppData below.

Comment (nopara73) [reply Stratis.Bitcoin/Configuration/NodeSettings.cs]: Just followed the existing code pattern. I agree it's simpler that way. Modified.

**Comment by bokobza** [comment]
Lovely, thank you.

**Pull request merged**